modules/DCService.js

```diff
@@ -94,7 +111,7 @@ class DCService {
             offer.save({ fields: ['status'] });
 
             const finalizationCallback = () => {
-                Models.offers.findOne({ where: { id: importId } }).then((offerModel) => {
+                Models.offers.findOne({ where: { import_id: importId } }).then((offerModel) => {
```

Comment:
I'm not sure this will guarantee which record will be returned. Imagine you have 3 offers with same importId, which record will be returned. Sequelize docs states that first record will be returned>
http://docs.sequelizejs.com/manual/tutorial/models-usage.html